<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parental Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/logo.ico</normaloff>:/icon/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog {background-color: rgb(221, 242, 253);}

QPushButton{
padding: 5px 20px 5px 20px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QFrame" name="ButtonFrame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: &quot;Roboto&quot;, sans-serif;
color: white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="monitor_setting_box">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
	color:black;
    spacing: 10px;
}

QCheckBox::indicator {
    width: 24px;
    height: 24px;
}

QCheckBox::indicator:unchecked {
    image: url(:/icon/icons8-toggle-off-48.png);
}

QCheckBox::indicator:checked {
    image: url(:/icon/icons8-toggle-on-48.png);
}
</string>
        </property>
        <property name="text">
         <string>Monitor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_parental_btn">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #a10000;
}

QPushButton:hover{
background: #d10000;
}</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restore_btn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {background-color: #00d991;}
QPushButton:hover{background-color: #00fca8;}
QPushButton:pressed{background-color: #00fca8;}</string>
        </property>
        <property name="text">
         <string>Restore Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exct_data_btn">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #427D9D;
padding: 5px 20px 5px 20px;
color: white;
}

QPushButton:hover{
background: #9BBEC8;
}
</string>
        </property>
        <property name="text">
         <string>Extract All Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="changePIN_btn">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #3b006e;
padding: 5px 20px 5px 20px;
}

QPushButton:hover{
background: #a200ff;
}</string>
        </property>
        <property name="text">
         <string>Change PIN</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
